frontend/src/components/Dashboard.js

```diff
@@ -27,26 +27,26 @@ export default function Dashboard({latitude, longitude, errorMessage}) {
             </DashboardBlock>
             <div>
                 <PrimaryButton labelButton={"Log Out"} />
-                <PrimaryButton labelButton={"Input"} handleClick={redirectToInput}/>
+                <PrimaryButton labelButton={"Trim Input"} handleClick={redirectToTrimInput}/>
             </div>
         </PageLayout>
     );
 
-    function redirectToInput() {
-        history.push("/input");
+    function redirectToTrimInput() {
+        history.push("/triminput");
     }
 }
 
 const PageLayout = styled.div`
 display: grid;
-grid-template-rows: 60px 120px 120px 1fr 60px;
+grid-template-rows: 60px 120px 1fr 60px;
```

Comment:
Try to use more fr or other relative units in grid templates, it'll serve you well for responsiveness in down the line =)